README.md

```diff
@@ -45,6 +45,21 @@ module.exports = {
 };
 ```
 
+To manually set the package name for publishing more elaborate Jenkins reports (instead of browser + version), just add a `'packageName'`:
+
+```js
+  // ...
+  reporters: ['dot', 'junit'],
+  reporterOptions: {
+    showDiff: true,
+      outputDir: './',
+      packageName: `${customPackageName}`
```

Comment:
What is `customPackageName` here? Can we replace that with a real world example? This looks rather confusing.